bluepyefe/ecode/step.py

```diff
@@ -79,8 +79,9 @@ def interpret(self, t, current, config_data, reader_data):
         # Smooth the current
         smooth_current = None
 
-        # Set the threshold to look for steps
-        epsilon = 0.005
+        # Set the threshold to detect the step
+        noise_level = numpy.std(numpy.concatenate((self.current[:50], self.current[-50:])))
```

Comment:
Are we sure that the step is always outside of the first 50 and last 50 points?

Reply:
(10 000Hz is the lowest sampling frequency used in electrophysiology. So 50 data point is at most 5ms.)
I cannot guarantee that is is always true, but it is for all of the traces I have ever seen.